doc/content/integrations/cloud-integrations/thingspeak/tts-setup/_index.md

```diff
@@ -4,25 +4,19 @@ description: ""
 weight: 2
 ---
 
-Once you have prepared the setup on ThingSpeak, follow this section to create a Webhook integration on {{% tts %}}.
+Once you have prepared the setup on ThingSpeak, follow this section to create a Webhook integration on {{% tts %}} by using the **ThingSpeak** [Webhook template]({{< ref "/integrations/webhooks/webhook-templates" >}}).
 
 <!--more-->
 
->**Note:** this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide.  
+>**Note:** To see the values of all parameters of the ThingSpeak integration, click on the integration after you create it with the Webhook template. In case you need to create a **Custom webhook**, see the [Creating Webhooks]({{< ref "/integrations/webhooks/creating-webhooks" >}}) guide.
```

Comment:
I would suggest moving this to the bottom and removing the last sentence, to avoid confusion